src/core/Akka.API.Tests/LogFormatSpec.cs

```diff
@@ -107,19 +107,28 @@ await AwaitConditionAsync(() =>
         // need to sanitize the thread id
         text = SanitizeDateTime(text);
         text = SanitizeThreadNumber(text);
+        // to resolve https://github.com/akkadotnet/akka.net/issues/7421
+        text = SanitizeTestEventListener(text);
         
         await Verifier.Verify(text);
     }
+    
+    private static string SanitizeTestEventListener(string logs)
+    {
+        var pattern = @"^.*Akka\.TestKit\.TestEventListener.*$";
+        var result = Regex.Replace(logs, pattern, string.Empty, RegexOptions.Multiline);
```

Comment:
Strip the offending `[DEBUG][DateTime][Thread 0001][EventStream(test)] Logger log1-TestEventListener [Akka.TestKit.TestEventListener]` line if the `TestEventListener` is detected.